goverge/main.py

```diff
@@ -93,7 +93,8 @@ def goverge(options):
 
     else:
         sub_dirs = [x[0] for x in os.walk(project_root)
-                    if "/." not in x[0] and "Godeps" not in x[0]]
+                    if "/." not in x[0] and "Godeps" not in x[0] and
```

Comment:
you could probably make this a bit cleaner with something like:
         sub_dirs = [x[0] for x in os.walk(project_root)
                            if x[0] not in ["/.", "Godeps", "vendor"]]